test/extensions/filters/network/mysql_proxy/BUILD

```diff
@@ -0,0 +1,73 @@
+licenses(["notice"])  # Apache 2
+
+load(
+    "//bazel:envoy_build_system.bzl",
+    "envoy_cc_test_library",
+    "envoy_package",
+)
+load(
+    "//test/extensions:extensions_build_system.bzl",
+    "envoy_extension_cc_test",
+)
+
+envoy_package()
+
+envoy_cc_test_library(
```

Comment:
nit: `envoy_extension_cc_test_library`?

Reply:
does that exist? I copied this from other test files

Reply:
Yeap. https://github.com/envoyproxy/envoy/blob/master/test/extensions/extensions_build_system.bzl#L4

You can add it at `load`.

```
load(
    "//test/extensions:extensions_build_system.bzl",
    "envoy_extension_cc_mock",
    "envoy_extension_cc_test",
    "envoy_extension_cc_test_library",
)
```